app/views/shared/views/_alerts_list.html.haml

```diff
@@ -88,6 +89,8 @@
                     = _("Action")
                   %th
                     = _("User")
+                  %th
+                    = _("Offending %{type}" % {:type => "{{$parent.$parent.item.hostType.split('::').pop()}}"})
```

Comment:
@ZitaNemeckova the bracket is incorrect `_("Offending %{type}")`

Reply:
@romanblanco Fixed. Thanks :)